src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Figure implements IFigure {
+    private String name;
```

Comment:
remove this field, it's enough to use name of the figure inside of draw() method directly